use-timescale/vpc.md

```diff
@@ -11,33 +11,41 @@ cloud_ui:
 only accessible through your secured AWS infrastructure. This reduces the potential 
 attack vector surface and improves security.
 
 The data isolation architecture that ensures a highly secure connection between your apps and 
-$CLOUD_LONG:
+$CLOUD_LONG is:
 
-<img class="main-content__illustration"
-src="https://assets.timescale.com/docs/images/tsc-vpc-architecture.svg"
-alt="The AWS Security Groups dashboard"/>
+![$CLOUD_LONG isolation architecture](https://assets.timescale.com/docs/images/tsc-vpc-architecture.svg)
 
-Your apps run inside your AWS Customer VPC, your services always run 
+Your customer apps run inside your AWS Customer VPC, your $CLOUD_LONG services always run 
 inside the secure $CLOUD_LONG $VPC. You control secure communication between apps in
-your VPC and your services using a dedicated Peering VPC. The AWS PrivateLink connecting
-$CLOUD_LONG $VPC to the dedicated Peering VPC gives the same level of protection as using a direct 
-AWS PrivateLink connection. It only enables communication to be initiated from your Customer VPC 
-to services running in the $CLOUD_LONG $VPC. $CLOUD_LONG cannot initiate communication with your VPC.
+your VPC and your $SERVICE_SHORTs using a dedicated Peering $VPC. The AWS PrivateLink connecting
+$CLOUD_LONG $VPC to the dedicated Peering $VPC gives the same level of protection as using a direct 
+AWS PrivateLink connection. It only enables communication to be initiated from your Customer $VPC 
+to services running in the $CLOUD_LONG $VPC. $CLOUD_LONG cannot initiate communication with your $VPC.
 
-To configure this secure connection, you first create the Peering VPC with 
+To configure this secure connection, you first create the $CLOUD_LONG Peering $VPC with 
 AWS PrivateLink in $CONSOLE. After you have accepted and configured the 
-peering connection to your Customer VPC, you use AWS Security Groups to 
-restrict the services in your Customer VPC that are visible to the Peering VPC.
-The last step is to attach individual services to the Peering VPC. 
-
-The number of VPCs you can attach to your project depends on your [pricing plan][pricing-plans]. If you 
-need more VPCs either contact contact [support@timescale.com](mailto:support@timescale.com) or change your
-pricing plan in $CONSOLE. Each $CLOUD_LONG VPC can have as many peering connections as you need.
+peering connection to your Customer $VPC, you use AWS Security Groups to 
+restrict the services in your Customer $VPC that are visible to the Peering $VPC.
+The last step is to attach individual services to the Peering $VPC. 
+
+* You define each $CLOUD_LONG $VPC on a [$CLOUD_LONG project level][project-members]. 
+* You can attach:
+  * Any number of Customer $VPCs to a $CLOUD_LONG $VPC.
+  * A $SERVICE_LONG to a single $CLOUD_LONG $VPC at a time.
+  
+      The $SERVICE_SHORT and $VPC must be in the same AWS region. To support multiple AWS regions, you need multiple 
```

Comment:
This is not 100% true: Even if the customer creates more Timescale VPCs in different regions, the service still is in a given region, and they won't be able to attach it to a VPC in a different region.
I would change it to: The service and Timescale VPC must be in the same AWS region, but you can peer a Customer VPC and Timescale VPC that are in different regions.  
(Please, rephrase as desired).